### PR TITLE
Add some art to run all

### PR DESCRIPTION
**Pull request by 2hdddg**

### CODE REVIEW

File: run_all.py [hunk 1]
```diff
@@ -61,11 +69,24 @@ def run():
...
+                print(c*scale, end='')
+            print('')
+    print('')
+
 
 if __name__ == "__main__":
     (temp_path, driver_repo_path, branch) = setup_environment()
 
     for driver in drivers:
+        print_art(driver.get("art", ""), 2)
```
Comment (bigmontz) [inline run_all.py]: You're already checking if art exists, so you don't need to set a default for art.

**Review by bigmontz — APPROVED** [review]
🧑‍🎨 🎨

File: run_all.py (hunk 1 above)
Comment (AndyHeap-NeoTech) [reply run_all.py]: I find the lines pleasing and the theme thought provoking. The palette is a little monochromatic for my taste though.

**Review by AndyHeap-NeoTech — APPROVED** [review]

File: run_all.py (hunk 1 above)
Comment (2hdddg) [reply run_all.py]: I'm not sure it is worth the effort to add colours.. As far as I know logs in TeamCity will not render that anyway ;-)

**Pull request merged**